packages/fxa-payments-server/src/routes/Subscriptions/ActionButton.tsx

```diff
@@ -0,0 +1,95 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { Localized } from '@fluent/react';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { Customer } from '../../store/types';
+
+export type ActionButtonProps = {
+  customer: Customer;
+  onRevealUpdateClick: () => void;
+  revealFixPaymentModal: () => void;
+};
+
+export const ActionButton = ({
+  customer,
+  onRevealUpdateClick,
+  revealFixPaymentModal,
+}: ActionButtonProps) => {
+  const {
+    billing_agreement_id,
+    payment_provider,
+    paypal_payment_error,
+  } = customer;
+
+  const stripeActionButton = (
+    <button
+      data-testid="reveal-payment-update-button"
+      className="settings-button"
+      onClick={onRevealUpdateClick}
+    >
+      <Localized id="pay-update-change-btn">
+        <span className="change-button" data-testid="change-button">
+          Change
+        </span>
+      </Localized>
+    </button>
+  );
+
+  const paypalActionButton = (
+    <Localized id="pay-update-change-btn">
+      <LinkExternal
+        data-testid="change-payment-update-button"
+        className="settings-button"
+        href={`https://www.paypal.com/myaccount/autopay/connect/${billing_agreement_id}`}
+      >
+        <span className="change-button" data-testid="change-button">
+          Change
+        </span>
+      </LinkExternal>
+    </Localized>
+  );
+
+  const paypalFundingSourceActionButton = (
+    <Localized id="pay-update-manage-btn">
+      <LinkExternal
+        data-testid="manage-payment-update-button"
+        className="settings-button error-button"
+        href={`https://www.paypal.com/myaccount/autopay/connect/${billing_agreement_id}`}
```

Comment:
We should note for QA that this link will look like an error when testing on Stage -- even if they change the domain manually after it opens in a new tab to `www.sandbox.paypal.com`. PayPal just doesn't support this webpage for sandbox accounts.

Reply:
Where would be a good spot to document this?

Reply:
I mentioned this in Slack as well, but I'd recommend we send QA an e-mail after this PR lands to let them know they can test this and list out any test cases that aren't yet captured in their existing test plan. We can work on that together if you'd like.